test/Lachain.CoreTest/IntegrationTests/BlocksTest.cs

```diff
@@ -260,6 +264,116 @@ public void Test_Wrong_Block_Emulation()
             Assert.AreEqual(OperatingError.InvalidMultisig, result);
         }
 
+        [Test]
+        public void Test_TxVerifierPerformanceCompare()
```

Comment:
Add Ignore attribute to this test for now please,  we should decrease the number of long unit tests and move them to the separate test project because we already near the timeout of the CI process

Reply:
done!